src/pages/recipes/index.md

```diff
@@ -451,3 +451,394 @@ function main(){
     }
 }
 ```
+
```

Comment:
This file is already pretty long. To make it readable please break it down into separate side nav items.

Reply:
And once you have separate files for each topic, please add search keywords.

Reply:
Agreed